### DISCUSSION TITLE
Support for ejs template syntax?

### DISCUSSION THREAD

**Discussion by Arooba-git** [babel/babel#15273] [Q&A] [answered]
Hello :) 
I'm using babel parser to traverse and parse angular/react projects.. (some of which have node backend repos as well)
when it encounters this node's template ejs syntax, it throws the 'unexpected token' error:
<img width="1240" alt="Screen Shot 2022-12-14 at 7 08 08 PM" src="https://user-images.githubusercontent.com/56495631/207574495-f831def4-5d2c-4825-9318-e6200e2165b8.png">

Is there any way to make babel ignore these files or not throw the error?

**Comment by liuxingbaoyu** [comment, marked as answer]
babel does not support ejs.
> Is there any way to make babel ignore these files or not throw the error?

It might be difficult not to throw exceptions, but you should have full control over which files are skipped parsing.

**Reply by Arooba-git** [reply]
Noted. Thanks for the swift response 👍
